Godeps/Godeps.json

```diff
@@ -243,19 +243,19 @@
 		},
 		{
 			"ImportPath": "github.com/shaleman/libOpenflow/common",
-			"Rev": "b692b84c96349d6f30d0cd2114196ef508847f9a"
+			"Rev": "94f338a137b50db6fa7d32e8cc3b406306616280"
 		},
 		{
 			"ImportPath": "github.com/shaleman/libOpenflow/openflow13",
-			"Rev": "b692b84c96349d6f30d0cd2114196ef508847f9a"
+			"Rev": "94f338a137b50db6fa7d32e8cc3b406306616280"
 		},
 		{
 			"ImportPath": "github.com/shaleman/libOpenflow/protocol",
-			"Rev": "b692b84c96349d6f30d0cd2114196ef508847f9a"
+			"Rev": "94f338a137b50db6fa7d32e8cc3b406306616280"
 		},
 		{
 			"ImportPath": "github.com/shaleman/libOpenflow/util",
-			"Rev": "b692b84c96349d6f30d0cd2114196ef508847f9a"
+			"Rev": "94f338a137b50db6fa7d32e8cc3b406306616280"
```

Comment:
@gkvijay Did you forget to godep update contivmodel changes? or was it already included by some other PR?